<?xml version="1.0"?>
<launch>

	<node name="robot_state_publisher" pkg="robot_state_publisher" exec="robot_state_publisher">
		<param name="robot_description" value="$(command 'xacro $(find-pkg-share neo_mpo_700-2)/robot_model/mpo_700/mpo_700.urdf')"/>
	</node>

	<!-- start node -->
	<node pkg="pilot_ros_bridge" exec="pilot_ros_bridge" name="pilot_ros_bridge">

		<param name="pilot_node" value="$(env HOME)/pilot/.pilot_main.sock"/>

		<param name="pilot_config" value="$(find-pkg-share pilot_ros_bridge)/config/default/mpo-700/"/>
	</node>

</launch>
